Makefile

```
@@ -1,6 +1,7 @@
+DOCKER_IMAGE = yellow:latest
 run:
-	docker run --rm -it -v `pwd`:/app -w /app rust:latest cargo run
+	docker run --rm -it -v `pwd`:/app -w /app ${DOCKER_IMAGE} cargo run
 build:
-	docker run --rm -it -v `pwd`:/app -w /app rust:latest cargo build --release
+	docker run --rm -it -v `pwd`:/app -w /app ${DOCKER_IMAGE} cargo build
 test:
-	docker run --rm -it -v `pwd`:/app -w /app rust:latest cargo test
+	docker run --rm -it -v `pwd`:/app -w /app ${DOCKER_IMAGE} cargo test
```

please add a `setup` option which builds the docker image so something like this:
```suggestion
	docker run --rm -it ${DOCKER_IMAGE} cargo test
setup:
	docker build -t yellow .
```